src/dealDashboard/discussionsList/discussionsList.html

```diff
@@ -23,7 +23,7 @@ <h1 class="subtitle heading heading1 gradient">
     <section if.to-view="!ethereumService.defaultAccountAddress">
       <p class="instructions">
         Only the deal proposal lead and representatives can participate in discussions.
-        <a class="connect-link" href="" click.delegate="this.ethereumService.ensureConnected()">Connect</a> to see if you are authorized to begin a discussion.
+        <a class="connect-link" href="" click.delegate="ethereumService.connect()">Connect</a> to see if you are authorized to begin a discussion.
```

Comment:
please use `ensureConnected`.  `connect` is private.

Reply:
done